editor/plugins/script_text_editor.cpp

```diff
@@ -1164,11 +1164,20 @@ void ScriptTextEditor::_update_connected_methods() {
 	// Add override icons to methods.
 	methods_found.clear();
 	for (int i = 0; i < functions.size(); i++) {
-		StringName name = StringName(functions[i].get_slice(":", 0));
+		String raw_name = functions[i].get_slice(":", 0);
+		StringName name = StringName(raw_name);
 		if (methods_found.has(name)) {
```

Comment:
Further down it's doing methods_found.insert on the shortened name. But up here you're comparing the full name, including period. I believe you'll need to tweak the insert to use the full name as well.